### ISSUE TITLE
How to build for windows

### ISSUE THREAD

**Issue by darkred** [NickeManarin/ScreenToGif#889]
Could someone please provide instructions about how to build this app? for windows 10 ?
I haven't found any info in this repo or https://www.screentogif.com/

What is the suggested workflow and prerequisites?

*I'd like to try to make a PR for my feature suggestion #842*

Thank you

**Comment by NickeManarin** [comment]
It's currently .NET Framework 4.8.
You can use Visual Studio 2019 to build.

**Comment by darkred** [comment]
Yes, I saw in the readme.md that ".NET Framework 4.8 and up is required", 
but since .NET Framework 4.8 was the final version, and now all future work is going into the .NET Core,
I was wondering what should I install, `.NET framework` or `.NET Core` ? 
(and if it's the latter, do I need the full `.NET Core SDK` or just `.NET Core Runtime` ?) .

And I I'd like to learn what tools are you using to develop such a program.
Do you use a full-fledged, IDE, such `Visual Studio`, or just a code editor, such as `VSCode` + its C# extension.

&nbsp;
I am a starter in C#. I don't want to turn this question issue into a "help me to learn C#", of course, 
I just want to learn what tools an experienced dev like you uses, in other words your suggested workflow.

**Comment by darkred** [comment]
Thank you very much for replying.
